cbits/encrypted_sign.c

```diff
@@ -253,9 +253,9 @@ static void serialize_index32(uint8_t *out, uint32_t index, derivation_scheme_mo
 
 static void add_left(ed25519_secret_key res_key, uint8_t *z, ed25519_secret_key priv_key, derivation_scheme_mode mode)
 {
-	uint8_t zl8[32];
+	ed25519_secret_key zl8;
```

Comment:
The `unit8_t` array had `32` elements (which is what caused the error message) and is replace with `ed25519_secret_key` which is 64 elements.

Reply:
Before the fix, the error was:
```
cbits/encrypted_sign.c:304:9: error:
     warning: ‘cardano_crypto_ed25519_publickey’ reading 64 bytes from a region of size 32 [-Wstringop-overread]
      304 |         cardano_crypto_ed25519_publickey(zl8, pub_zl8);
          |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    |
304 |         cardano_crypto_ed25519_publickey(zl8, pub_zl8);
    | 
```